test/tests/api/redfish_1_0/event_service_tests.py

```diff
@@ -86,19 +86,28 @@ def test_get_event_subscription(self):
     def test_submit_test_event(self):
         """ Testing POST /EventService/SubmitTestEvent  """
         global task
-        server = Httpd(port=int(HTTPD_PORT), handler_class=self.EventServiceHandler)
+        # Suppose rackhd and test stack have the same localhost
```

Comment:
@hohene Sorry, this feature was created before tests/api-cit/redfish_1_0 exists.

I've updated that case.